### ISSUE TITLE
contrast of font color on white background is too low

### ISSUE THREAD

**Issue by ddan39** [frappe/erpnext#15199]
The font colors in many places are way too light for a white background. It is difficult to read the text. This has been a complaint from many people I try to introduce ERPNext to. I am not saying I want pure black on white, I just think the color needs to be much darker, like #333 maybe.

For example, on the one page I was viewing there is color #8D99A6 light blueish grey text on pure white #FFF background, that is a contrast ratio of 2.9:1. This contrast ratio is way below the absolute minimum ratio in W3C, ANSI, and ISO standards.

I honestly don't know how anyone uses erpnext on a daily basis with the layout/color schemes the way they are. I get that it is the latest trend to have the text smaller and lighter colored, but it is just not practical.

Thanks,
Dan

**Comment by Strider27** [comment]
I agree, its fairly hard to see, especially on certain screens. My solution for now is to use "Dark Reader" chrome addon.

**Comment by strixaluco** [comment]
As @Strider27 said, it depends on a screen, although I agree it's easier to change the font color than to buy a better display.

**Comment by system19** [comment]
There should be a normal/darker text contrast setting in the user drop down menu. A normal/larger text setting would be good too.

**Comment by PhotoBiker** [comment]
I'm a newbie to ERPnext, just got it to work on a laptop today for testing. I just wanted to add my 2 cents here that I totally agree that it's really hard to read.
I think it would be a great idea if ERPnext would allow for themes/ templates (similar to WordPress). This way, folks could edit the templates, and probably the community would come up with some nifty solutions :)

**Comment by systemdarena** [comment]
Hello, any updates on this? Are there any easy ways to change this in version 11 now?

It honestly seems like this is a core issue with someone leading the entire ERPNext project, since I noticed the erpnext sites also follows this fad.

**Comment by Salmon-Bard** [comment]
We have elder employees with weak vision
There should be option to improve text readability
